contracts/AceArcade.vy

```diff
@@ -0,0 +1,52 @@
+# @version ^0.3.10
+# Data structuress
+game_id: public(uint256)
+games: public(HashMap[uint256, HashMap[address, bytes32]])
+player_hands: public(HashMap[uint256, HashMap[address, uint256[2]]])
+computer_hands: public(HashMap[uint256, uint256[2]])
+
+@internal
+@view
+def getAccountBalance(player: address) -> uint256:
+    # This will be handled in another contract
+    # Will return the value of the user's account
+    return 10000
+
+@external
+@payable
+def startGame(player: address, bet: uint256):
+    # Check if player has sufficient balance
+    bal : uint256 = self.getAccountBalance(player) 
+    assert bal >= bet, "Insufficient balance"
+
+    # Create a new game with a unique game ID
+    self.game_id += 1
+    game_id: uint256 = self.game_id  # Local variable
+
```

Comment:
Add a TODO - that says link to RandomNumber when it exists.  Make it HUGE

****
***  TODO - LINK TO RANDOM NUMBER GENERATOR ***
****

Reply:
tests/test_aceArcade.py .                                                                                        [ 20%]
tests/test_exampleGame.py ...                                                                                    [ 80%]
tests/test_mintToken.py .                                                                                        [100%]

================================================== 5 passed in 0.98s ===================================================